### PR TITLE
Makefile fixes

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,72 +1,80 @@
...
+GO111MODULE			:= off
+KUBEBUILDER_ASSETS	:= $(CURDIR)/bin/deps
+PATH				:= $(CURDIR)/bin/deps:$(CURDIR)/bin/deps/go/bin:$(PATH)
+GOROOT				:= $(CURDIR)/bin/deps/go
+GOVERSION			:= $(shell go version)
 
 # Run tests
+.PHONY: test
 test: fmt vet vendor
-	./bin/deps/go/bin/go test ./pkg/... ./cmd/... -coverprofile cover.out
```
Comment (dahankzter) [inline Makefile:19]: If we don't use the downloaded Go anymore we should also avoid downloading it.

Comment (mmatczuk) [reply Makefile:19]: We use it's in path.

Comment (dahankzter) [reply Makefile:19]: Yes but there is a script that downloads it to `bin/deps` and that seems wasteful now.

Comment (mmatczuk) [reply Makefile:19]: It is what it is we can remove it in another PR, this works with both project and local go.

Comment (dahankzter) [reply Makefile:19]: Will it work? I am not sure how the github actions do it to run unit tests.

Comment (dahankzter) [reply Makefile:19]: It seems to run fine. :)

**Pull request merged**